search/variable_v2.go

```diff
@@ -188,3 +263,27 @@ func LoadVariableTranslationsFromFile(variableFile string, variableName string)
 
 	return translations, nil
 }
+
+func GenerateSourcePositionVariables(db *sql.DB) (TranslationsV2, error) {
+	translations := make(TranslationsV2)
+	query := `select max(position) from sources`
+	var max int
+	err := queries.Raw(db, query).QueryRow().Scan(&max)
+	if err != nil {
+		return nil, errors.Wrap(err, "Unable to retrieve max sources position from DB.")
+	}
+	for _, lang := range consts.ALL_KNOWN_LANGS {
+		values := make(map[string][]string)
+		for i := 1; i < max+1; i++ {
+			numStr := strconv.Itoa(i)
+			values[numStr] = []string{
+				numStr,
```

Comment:
Why array o one element?

Reply:
Line after we optionally add another element to the array (numeric presentation in Hebrew)